xbmc/windowing/egl/EGLWrapper.cpp

```diff
@@ -22,6 +22,8 @@
 #ifdef HAS_EGL
 #include "utils/log.h"
 #include "EGLNativeTypeAndroid.h"
+#include "EGLNativeTypeAmlAndroid.h"
+#include "EGLNativeTypeRKAndroid.h"
```

Comment:
Totally agree. Will do a "pre-commit" with proper #ifdefs. No point to have Wayland for Android, either ;)